gio/src/auto/file.rs

```diff
@@ -675,7 +675,7 @@ pub trait FileExt: 'static {
 
     #[doc(alias = "g_file_resolve_relative_path")]
     #[must_use]
-    fn resolve_relative_path(&self, relative_path: impl AsRef<std::path::Path>) -> Option<File>;
+    fn resolve_relative_path(&self, relative_path: impl AsRef<std::path::Path>) -> File;
```

Comment:
Should also make a bugfix release with this. Previous bugfix release changed this to an `Option`, which was apparently wrong.